### DISCUSSION TITLE
My Solidity Journey

### DISCUSSION THREAD

**Discussion by Mr-Nep** [Cyfrin/foundry-full-course-cu#1460] [General]
Today 5th March, I want to start enjoying the great work of Cyrin and Co for the Superb Tutorial, i have seen lovely comments from the community and i know am on the right track

**Comment by EngrPips** [comment]
@Mr-Nep  Indeed you are on the right track and don't hesitate to always reach out on this github discussion channel anytime you get stuck or couldn't figure out something

**Reply by Mr-Nep** [reply]
Thanks 
i would definitly reach out for assistance if i get stuck
